invenio/modules/knowledge/api.py

```diff
@@ -321,15 +334,15 @@ def add_kb(kb_name="Untitled", kb_type=None, tries=10):
             name = kb_name + " " + str(index)
             i = i + 1
             created = False
-        except:
+        except Exception:
             db.session.rollback()
             raise
 
     if created is False:
         # TODO raise the right exception
-        raise Exception("Can't create kb \"{0}\".\n" +
-                        "Probabily the server is busy! " +
-                        "Try again later.".format(kb_name))
+        raise Exception(_("Can't create kb \"{0}\".\n" +
```

Comment:
IMHO this should/does not have to be translated. Btw inside bracket you don't need `+` for string concatenation.